src/Tribe/Attendee_Registration/View.php

```diff
@@ -21,16 +21,16 @@ public function __construct() {
 	 *
 	 * @since 4.9
 	 * @param string $content The original page|post content
-	 * @return srting $template The resulting template content
+	 * @param string $context The context of the rendering
+	 * @return string $template The resulting template content
```

Comment:
`@return` doesn't get a $varname

```suggestion
	 * @return string The resulting template content
```